book/src/SUMMARY.md

```diff
@@ -36,6 +36,7 @@
   - [Data Plane Fanout](data-plane-fanout.md)
   - [Reliable Vote Transmission](reliable-vote-transmission.md)
   - [Bank Forks](bank-forks.md)
+  - [Leader to Leader Handoff](leader-leader-handoff.md)
```

Comment:
Your other proposal had a very similar title "Leader to Validator Transition" with filename "leader-validator-transition.md". Here you use the term "handoff". Why the difference?

Reply:
@garious I could use the same term.  The leader to validator transition was the transition inside the fullnode.  This transition is covering how each leader hands-off PoH production to the next one.

Reply:
Doesn't matter. Just use the same word consistently.